app/models/public_body.rb

```diff
@@ -591,8 +591,8 @@ def json_for_api
       # information
       # :version, :last_edit_editor, :last_edit_comment
       :home_page => calculated_home_page,
-      :notes => notes,
-      :publication_scheme => publication_scheme,
+      :notes => notes.to_s,
+      :publication_scheme => publication_scheme.to_s,
```

Comment:
Why? Presumably because `nil` gets set as `null`?

```ruby
{ :a => nil }.to_json
# => "{\"a\":null}"
{ :a => '' }.to_json
# => "{\"a\":\"\"}"
```

Should have specs to document this.